views/applications/all.html

```diff
@@ -0,0 +1,47 @@
+{% extends "_donut.html" %}
+
+{% block pageHead %}
+  {{ super() }}
+{% endblock %}
+
+{% block header %}
+  {{ super() }}
+{% endblock %}
+
+{% block main %}
+  <div class="introduction">
+    <h1 class="introduction--h1">{{structure.name}}</h1>
+    <p class="introduction--p">{{structure.description}}</p>
+    <a href="/{{config.base}}/{{config.actions.add}}" class="introduction--button">{{config.actions.add|capitalize}} {{config.base}}</a>
+  </div>
+
+  {% if applications %}
+  <table class="base--table">
+    <thead class="base--thead">
+      <tr class="base--tr">
+        <th class="base--th">Application</th>
+        <th class="base--th">Live</th>
+        <th class="base--th">Updated</th>
+        <th class="base--th">Sunset</th>
+        <th class="base--th">Edit</th>
+      </tr>
+    </thead>
+    <tbody class="base--tbody">
+    {% for app in applications %}
+      <tr class="base--tr">
+        <td data-th="Application" class="base--td">{{app.name}}</td>
+        <td data-th="Live" class="base--td">{% if app.endpoints.live %}<a href="/{{config.base}}/{{a.id}}#live" class="base--link"><span class="endpoint-{{app.endpoints.live|first}}">{{app.endpoints.live|first}}</span></a>{% endif %}</td>
```

Comment:
Are you expecting to have the same data structure here as you are in `one.html`? If so, these need to be grabbing `response` property of the first item, not just the first item.